packages/transformers/js/src/JSTransformer.js

```diff
@@ -66,7 +66,6 @@ export default new Transformer({
   },
 
   async transform({asset, options}) {
-    asset.type = 'js';
```

Comment:
🤔

Reply:
I'm not using `.jsx` anymore in the tests and have added this bug to the Todo.